packages/web/src/js/FileUploader.ts

```diff
@@ -369,7 +387,8 @@ class FileUploader extends BaseComponent {
     }, 0);
   }
 
-  static onDragOver(event: Event) {
+  // eslint-disable-next-line class-methods-use-this
+  onDragOver(event: Event) {
```

Comment:
Why is here this change? This method is not somehow static or does not work correctly?

Reply:
I'll put it back; I forgot this from my first demo implementation. This change is not necessary anymore. Thanks